src/components/Project.js

```diff
@@ -44,46 +60,67 @@ class Project extends Component {
   render() {
     const imageURI = `https://${this.props.project.avatar_src}`
```

Comment:
It would be good to have a default image here if there isn't one for `project.avatar_src`. The teal_wallpaper.png image above would probably work.